src/base/bittorrent/torrenthandle.cpp

```diff
@@ -190,7 +192,11 @@ TorrentState::operator int() const
 const qreal TorrentHandle::USE_GLOBAL_RATIO = -2.;
 const qreal TorrentHandle::NO_RATIO_LIMIT = -1.;
 
+const int TorrentHandle::USE_GLOBAL_SEEDING_TIME = -2;
+const int TorrentHandle::NO_SEEDING_TIME_LIMIT = -1;
+
 const qreal TorrentHandle::MAX_RATIO = 9999.;
+const int TorrentHandle::MAX_SEEDING_TIME = 525600;
```

Comment:
I have to put a number in there, like the max ratio has one.  I chose the first coherent option I could think of.